packages/victory-line/src/victory-line.test.tsx

```diff
@@ -322,11 +314,11 @@ describe("components/victory-line", () => {
       );
       const path = container.querySelector("path");
 
-      expect(path.getAttribute("aria-label")).toEqual(
+      expect(path?.getAttribute("aria-label")).toEqual(
         `data point 3's x value is 2`,
       );
 
-      expect(parseInt(path.getAttribute("tabindex"))).toEqual(3);
+      expect(parseInt(path?.getAttribute("tabindex") || "")).toEqual(3);
```

Comment:
Conversion looks good!  Merge away.

Possible improvement: switch to `!` instead of `?` when in tests, because error messages would be better.  Like `cannot access 'getAttribute' of null` versus `expected "" to equal 3`